--- a/install/installcommand.cpp
+++ b/install/installcommand.cpp
@@ -32,13 +32,8 @@
 #include "otautil/print_sha1.h"
 
 bool verify_package_compatibility(ZipWrap *zw) {
-#ifdef OF_NO_TREBLE_COMPATIBILITY_CHECK
-   printf("Bypassing Treble compatibility check ...\n");
+#   printf("OrangeFox: Forcefully bypassing Treble compatibility check via patch.\n");
    return true;
-#endif
-  ZipArchiveHandle package_zip = zw->GetZipArchiveHandle();
-  printf("Verifying package compatibility...\n");
 
   static constexpr const char* COMPATIBILITY_ZIP_ENTRY = "compatibility.zip";
   ZipString compatibility_entry_name(COMPATIBILITY_ZIP_ENTRY);
