--- a/installcommand.cpp
+++ b/installcommand.cpp
@@ -348,13 +348,8 @@
 // Verifes the compatibility info in a Treble-compatible package. Returns true directly if the
 // entry doesn't exist. Note that the compatibility info is packed in a zip file inside the OTA
 // package.
-bool verify_package_compatibility(ZipWrap *zw) {
-#ifdef OF_NO_TREBLE_COMPATIBILITY_CHECK
-   printf("Bypassing Treble compatibility check ...\n");
+bool verify_package_compatibility(ZipWrap *zw) { // NOLINT
+   printf("OrangeFox: Forcefully bypassing Treble compatibility check via patch.\n");
    return true;
-#endif
-  ZipArchiveHandle package_zip = zw->GetZipArchiveHandle();
-  printf("Verifying package compatibility...\n");
 
   static constexpr const char* COMPATIBILITY_ZIP_ENTRY = "compatibility.zip";
   ZipString compatibility_entry_name(COMPATIBILITY_ZIP_ENTRY);
